src/main/java/Frappe.java

```diff
@@ -0,0 +1,308 @@
+            throw new FrappeException(FrappeException.TOO_MANY_TO);
+        }
+
+        String name = input[0].trim();
+        String from = input2[0].trim();
+        String to = input2[1].trim();
+
+        if (name.isEmpty()) {
+            throw new FrappeException(FrappeException.NO_NAME);
+        }
+
+        if (from.isEmpty()) {
+            throw new FrappeException(FrappeException.NO_FROM);
+        }
+
+        Task task = new Event(name, from, to);
+        tasks.add(task);
+        Printer.printTaskAdded(task);
+    }
+
+    private static void removeTask(String[] words) throws FrappeException {
+        int index = getTaskIndex(words);
+        Task task = tasks.get(index);
+        tasks.remove(task);
+        Printer.printTaskRemoved(task);
+    }
+
+    private static void processInput(String[] words) throws FrappeException {
+        switch (words[0]) {
+        case "list":
+            Printer.printTasks();
+            break;
+        case "mark":
+            updateTaskDone(words, true);
+            saveTasks();
+            break;
+        case "unmark":
+            updateTaskDone(words, false);
+            saveTasks();
+            break;
+        case "todo":
+            addTodo(words);
+            saveTasks();
+            break;
+        case "deadline":
+            addDeadline(words);
+            saveTasks();
+            break;
+        case "event":
+            addEvent(words);
+            saveTasks();
+            break;
+        case "delete":
+            removeTask(words);
+            saveTasks();
+            break;
+        default:
+            throw new FrappeException(FrappeException.UNKNOWN_COMMAND);
+        }
+    }
+
+    private static void saveTasks() throws FrappeFileException {
+
+        try {
+            File saveFile = new File(saveFilePath);
+            saveFile.getParentFile().mkdirs();
+            saveFile.createNewFile();
+
+            FileWriter writer = new FileWriter(saveFilePath);
+
+            for (Task task : tasks) {
+                writer.write(task.getTypeString());
+                writer.write("\n");
+                writer.write((task.getDone() ? "1" : "0"));
+                writer.write("\n");
+                writer.write(task.getName());
+                writer.write("\n");
+
+                if (task instanceof Deadline) {
+                    writer.write(task.getDoBy());
+                    writer.write("\n");
+                } else if (task instanceof Event) {
+                    writer.write(task.getFrom());
+                    writer.write("\n");
+                    writer.write(task.getTo());
+                    writer.write("\n");
+                }
+
+                writer.write("\n");
+            }
+            writer.close();
+        } catch (IOException e) {
+            throw new FrappeFileException(FrappeFileException.SAVE_FAILED_WARNING);
+        }
+    }
+
+    private static void loadTasks() throws FrappeFileException {
```

Comment:
This method is quite long and a bit hard to read in my opinion. Perhaps abstracting some of the inner methods would help?